systemtest/src/test/java/io/strimzi/systemtest/operators/FeatureGatesIsolatedST.java

```diff
@@ -344,4 +347,75 @@ public void testKRaftMode(ExtensionContext extensionContext) {
         LOGGER.info("Waiting for clients to finish sending/receiving messages.");
         ClientUtils.waitForClientsSuccess(producerName, consumerName, INFRA_NAMESPACE, MESSAGE_COUNT);
     }
+    @IsolatedTest
+    void testSwitchingConnectStabilityIdentifiesFeatureGateOnAndOff(ExtensionContext extensionContext) {
+        assumeFalse(Environment.isOlmInstall() || Environment.isHelmInstall());
+
+        final TestStorage testStorage = new TestStorage(extensionContext);
+        final int connectReplicas = 1;
+        final int messageCount = 500;
+        List<EnvVar> coEnvVars = new ArrayList<>();
+
+        coEnvVars.add(new EnvVar(Environment.STRIMZI_FEATURE_GATES_ENV, "", null));
+
+        LOGGER.info("Deploying CO without CSI");
+
+        clusterOperator.unInstall();
+        clusterOperator = new SetupClusterOperator.SetupClusterOperatorBuilder()
+                .withExtensionContext(BeforeAllOnce.getSharedExtensionContext())
+                .withNamespace(testStorage.getNamespaceName())
+                .withWatchingNamespaces(Constants.WATCH_ALL_NAMESPACES)
+                .withExtraEnvVars(coEnvVars)
+                .createInstallation()
+                .runInstallation();
+
+        resourceManager.createResource(extensionContext, KafkaTemplates.kafkaPersistent(testStorage.getClusterName(), 3, 1).build());
+        resourceManager.createResource(extensionContext, KafkaTopicTemplates.topic(testStorage.getClusterName(), testStorage.getTopicName()).build());
+        resourceManager.createResource(extensionContext, KafkaConnectTemplates.kafkaConnect(testStorage.getClusterName(), testStorage.getNamespaceName(), connectReplicas).build());
+
+        Map<String, String> coPod = DeploymentUtils.depSnapshot(clusterOperator.getDeploymentNamespace(), ResourceManager.getCoDeploymentName());
+
+        final LabelSelector connectLabelSelector = KafkaConnectResource.getLabelSelector(testStorage.getClusterName(), KafkaConnectResources.deploymentName(testStorage.getClusterName()));
+        Map<String, String> connectPods = PodUtils.podSnapshot(testStorage.getNamespaceName(), connectLabelSelector);
+
+        KafkaClients clients = new KafkaClientsBuilder()
+                .withProducerName(testStorage.getProducerName())
+                .withConsumerName(testStorage.getConsumerName())
+                .withBootstrapAddress(KafkaResources.plainBootstrapAddress(testStorage.getClusterName()))
+                .withTopicName(testStorage.getTopicName())
+                .withMessageCount(messageCount)
+                .withDelayMs(1000)
+                .withNamespaceName(clusterOperator.getDeploymentNamespace())
+                .build();
+
+        resourceManager.createResource(extensionContext,
+                clients.producerStrimzi(),
+                clients.consumerStrimzi()
+        );
+
+        LOGGER.info("Changing FG env variable to enable SCI");
```

Comment:
I would use a full name of the feature gate to make it more clear what does it mean.